### PR TITLE
Sort charts

### PR DESCRIPTION
**Pull request by seanebum**

### CODE REVIEW

File: src/reducers/analyzeAreaSlice.js [hunk 1]
```diff
@@ -30,11 +38,17 @@ export const AnalyzeAreaSlice = createSlice({
...
+    },
     changeSortDirection: (state, action) => {
       state.isSortASC = !state.isSortASC;
     },
+    changeChartSortDirection: (state, action) => {
+      state.chartSortASC = !state.chartSortASC;
+    },
     changeSortBy: (state, action) => {
-      state.sortBy = action.payload;
+      state.sortBy = (action.payload !== state.sortBy) ? action.payload : null;
```
Comment (jeffbliss) [inline src/reducers/analyzeAreaSlice.js:51]: I was observing some weird behavior with this... e.g. if you click on "Threat" and throw some logs of action.payload and state.sortBy in there - you can see that state.sortBy becomes null at some point.

I changed this to `state.sortBy = action.payload;` and it seems to work as expected. Just wanted to make sure there wasn't something I'm missing

Comment (jeffbliss) [reply src/reducers/analyzeAreaSlice.js:51]: Oh I see. It's allowing you to remove the sorting.

File: src/components/AnalyzeArea/ChartSortHolder.js
Comment (jeffbliss) [inline src/components/AnalyzeArea/ChartSortHolder.js:1]: I'm not sure I can figure out what className, communityExposureClassName etc etc are doing? I don't see them as props in ChartSort and I actually removed them and don't see any discernible difference?

**Comment by jeffbliss** [comment]
High level comment not sure where to put it:

I'm wondering if it would make more sense to remove the up/down arrow sort button and just allow the user to click on each element and that'll change whether it's sorting asc/desc per that element. 

We'd probably need to add a way to remove sorting all together if we went that route.

File: src/components/AnalyzeArea/ChartSortHolder.js
Comment (seanebum) [reply src/components/AnalyzeArea/ChartSortHolder.js:1]: oh these can come out!  They were a holdover from playing with the figma tool!  Good eye

**Pull request merged**